en/docs/developer-portal/manage-subscription.md

```diff
@@ -0,0 +1,20 @@
+# Subscribe to an API
+
+A subscription to an API makes it possible for the application to to know which APIs it can authenticate with it's keys. You have to subscribe the **published** API  to an application to invoke it using the application credentials.
+
+Alternatively, you can generate credentials for an API without an explicit subscription to an application. However, this will not let you control advanced configurtaion such as access token expiry time, revoke token expiry time, ID token token expiry time, and enabling access to the API without a secret. Generating keys in the API is recommended for testing or short-term usage but not for long-term production usage. 
```

Comment:
```suggestion
Alternatively, you can generate credentials for an API without an explicit subscription to an application. However, this will not let you control advanced configuration such as access token expiry time, revoke token expiry time, ID token expiry time, and enabling access to the API without a secret. Generating keys in the API is recommended for testing or short-term usage but not for long-term production usage. 
```